<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1407</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : &quot;white&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(204, 204, 204), stop:1 rgb(230, 230, 230))</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="input_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Input Device</string>
     </property>
    </widget>
    <widget class="QLabel" name="output_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Output Device</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_in">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(230, 230, 230)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_out">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>35</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(230, 230, 230)
</string>
     </property>
    </widget>
    <widget class="QLabel" name="output_lbl_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Sampling Rate (&gt;1000 Hz)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sampleRate_lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :&quot;white&quot;</string>
     </property>
     <property name="text">
      <string>44100</string>
     </property>
    </widget>
    <widget class="QLabel" name="output_lbl_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>selecting window Start (s)</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_xlimStart">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>85</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(230, 230, 230)</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_xlimEnd">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>85</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(230, 230, 230)</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="output_lbl_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>85</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>selecting window End (s)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="plot_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>581</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>plot</string>
     </property>
    </widget>
    <zorder>sampleRate_lineEdit</zorder>
    <zorder>output_lbl_3</zorder>
    <zorder>comboBox_out</zorder>
    <zorder>output_lbl</zorder>
    <zorder>input_lbl</zorder>
    <zorder>comboBox_in</zorder>
    <zorder>output_lbl_4</zorder>
    <zorder>doubleSpinBox_xlimStart</zorder>
    <zorder>doubleSpinBox_xlimEnd</zorder>
    <zorder>output_lbl_5</zorder>
    <zorder>plot_btn</zorder>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>300</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(204, 204, 204), stop:1 rgb(230, 230, 230))
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Recording Time(s)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_recordingTime">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>35</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(230, 230, 230)</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>YRange</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_yrangemin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(230, 230, 230)</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>0.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>-0.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_yrangemax">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(230, 230, 230)</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="play_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_btn">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>110</y>
       <width>70</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="record_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>70</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Sound Defalt chaneels</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_num_channels">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>85</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(230, 230, 230)</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>dtype</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_dtype">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>int 8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>int 16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>int 32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>float</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>901</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>601</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_Spectrogram">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>881</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_1"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_pre_emp">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>550</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>0</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>240</y>
      <width>501</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>500</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <zorder>frame</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>frame_2</zorder>
   <zorder>gridLayoutWidget_2</zorder>
   <zorder>gridLayoutWidget_3</zorder>
   <zorder>gridLayoutWidget_5</zorder>
   <zorder>gridLayoutWidget_6</zorder>
   <zorder>gridLayoutWidget_7</zorder>
   <zorder>gridLayoutWidget_8</zorder>
   <zorder>gridLayoutWidget_9</zorder>
   <zorder>gridLayoutWidget_10</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1407</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>Developer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
